.github/workflows/ci.yaml

```diff
@@ -1,4 +1,7 @@
-name: slurmdbd charm tests
+# Copyright 2023 Canonical Ltd.
+# Apache Software License, version 2.0
```

Comment:
I have it as `Apache Software License, version 2.0` because we cannot guarantee that the license of the Omnivector charms will stay the same in the future. Let's say that a few years down the line, whoever owns the slurm charms decides to change the license from Apache-2.0 to AGPLv3. If we have it as `See LICENSE file for licensing details`, that will effectively convince other devs that the file is licensed under AGPLv3 when it is actually licensed under Apache-2.0 by Canonical.

Perhaps @wolsen knows what format we should use for the license header in files contributed by Canonical to an external project?

Reply:
Should use the full-header here mentioned on the Apache website: https://www.apache.org/licenses/LICENSE-2.0#apply

Reply:
Resolved in https://github.com/canonical/slurmctld-operator/pull/1/commits/6fa8296081749b73897e3c2c72bbff5737f99322